### ISSUE TITLE
Many errors

### ISSUE THREAD

**Issue by Smitty010** [brianpetro/obsidian-smart-connections#825]
I get hundreds of the following errors:

import @ plugin:smart-connections:7022
  "TypeError: Cannot read properties of undefined (reading 'stat')\n    at SmartSource.import (plugin:smart-connections:7001:21)\n    at eval (plugin:smart-connections:7538:83)\n    at Array.map (<anonymous>)\n    at SmartSources.process_import_queue (plugin:smart-connections:7538:58)\n    at async SmartSources.process_load_queue (plugin:smart-connections:7528:7)\n    at async _SmartEnv.load_collections (plugin:smart-connections:5482:9)\n    at async _SmartEnv.load_main (plugin:smart-connections:5468:5)\n    at async _SmartEnv.init (plugin:smart-connections:5434:5)\n    at async _SmartEnv.create (plugin:smart-connections:5421:18)\n    at async SmartConnectionsPlugin.load_env (plugin:smart-connections:13941:5)\n    at async SmartConnectionsPlugin.initialize (plugin:smart-connections:13922:5)"

I also notice that there are no blocks in the ajson files in .smart_env/multi

**Comment by brianpetro** [comment]
Screenshots of the errors are most helpful 🌴

**Comment by Smitty010** [comment]
There is an error in the initial comment.  Is this better?
![image](https://github.com/user-attachments/assets/fda87519-fa9a-4c7f-bc9b-d3601192b6a1)

Also, it looks like you do thing in two step.  I let it run all night and it's now getting embeddings.  I have a large vault (16,000+) and it's taking awhile, but it is getting embeddings now.  I suspect it will finish that process later today.

**Comment by brianpetro** [comment]
Yeah, that helps, screenshots give more context and are easier to read than copy&pasted text.

It's probably a file type bug that's appeared in a few other recent issues. 

If you figure out a specific file type is causing the issue, let me know because that would help me fix the bug.

Either way, I will aim to make ensure unexpected file types don't throw errors and stop the processing like this before the next release
 🌴

**Comment by Smitty010** [comment]
It would be nice if it logged the offending file to the developer log or in
the exception as well.  I looked where it had stopped in my directory and
there wasn't anything that stuck out as to why it would stop there.  They
looked like ordinary files.
